src/components/Highlights.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="highlightsbar">
+    <span id="center-highlight">
+      {{ industryindices[highlightsindex1] }}
+      {{ industrybetas[highlightsindex1] }}
+    </span>
+    <span id="left-highlight"> JSE Allshare Index: {{ jseindex }}</span>
+    <span id="right-highlight">
+      {{ industryindices[highlightsindex2] }}
+      {{ industrybetas[highlightsindex2] }}
+    </span>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Highlights',
+  data() {
+    return {
+      industrybetas: ['0.7276 (5%)', '0.9 (-7%)', '1.1 (25%)', '0.68 (-30%)'],
```

Comment:
it's good practice to use camelCase for variable names in Javascript so this should be industryBetas